toss_4_x86_64_ib_cray/packages.yaml

```diff
@@ -0,0 +1,86 @@
+packages:
+  all:
+    # This defaults us to machine specific flags of ivybridge which allows
+    # us to run on broadwell as well
+    target: [zen2]
+    compiler: [gcc, cce, clang]
+  cmake:
+    version: [3.14.5]
+    buildable: false
+
+    externals:
+    - spec: cmake
+      prefix: /usr/tce/packages/cmake/cmake-3.14.5
+  cuda:
+    version: [11.4.120]
+    buildable: false
+
+    externals:
+    - spec: cuda@11.4.120
+      prefix: /opt/toss/cudatoolkit/11.4/ 
```

Comment:
Is this path ok? /opt/toss/cudatoolkit/11.4/bin/nvcc is a valid path, it just looks very different from the other cuda prefix paths..

Reply:
Sure - I'm not ever sure people will be using it.